data/raw/USB6014.txt

```diff
@@ -0,0 +1,158 @@
+1	2	277.1
```

Comment:
We might want to comment somewhere what this represents, someone coming into the project might not understand what `USB6014` means